NEWS

```diff
@@ -5,89 +5,145 @@ NEWS
 - [#5995](https://github.com/mruby/mruby/issues/5995) One seemingly unnecessary parameter is passed in the block parameters
+- [#6008](https://github.com/mruby/mruby/pull/6008) Make "bintest" independent of directory
+- [b47c8b7](https://github.com/mruby/mruby/commit/b47c8b7) gc.c (clear_all_old): fix a generational GC bug
 - [#6029](https://github.com/mruby/mruby/issues/6029) mruby build fails under mrbgems directory
+- [a264965](https://github.com/mruby/mruby/commit/a264965) mruby-os-memsize/memsize.c: fix irep size calculation
+- [3310e10](https://github.com/mruby/mruby/commit/3310e10) mruby-test/mrbgem.rake: fix mrb_state handling bug
 - [#6041](https://github.com/mruby/mruby/issues/6041) GC Performance may have degraded
+- [#6044](https://github.com/mruby/mruby/issues/6044) Generated presym/table.h contains invalid characters
 - [#6051](https://github.com/mruby/mruby/issues/6051) Null pointer dereference in mrb_addrinfo_unix_path
 - [#6052](https://github.com/mruby/mruby/issues/6052) Null pointer dereference while handling the Proc class
+- [#6055](https://github.com/mruby/mruby/pull/6055) Fix libmruby name for VisualC++
+- [#6060](https://github.com/mruby/mruby/issues/6060) SEGFAULT Issue Related to Fiber Usage in ngx_mruby Development
+- [#6061](https://github.com/mruby/mruby/issues/6061) Performance issue in String#codepoints
+- [#6064](https://github.com/mruby/mruby/issues/6064) MRUBY_PACKAGE_DIR does not always have a value.
 - [#6065](https://github.com/mruby/mruby/issues/6065) Null pointer dereference while handling the Proc class
 - [#6066](https://github.com/mruby/mruby/issues/6066) Null pointer dereference involving Struct.new()
 - [#6067](https://github.com/mruby/mruby/issues/6067) Null pointer dereference in mrb_string_value_cstr
 - [#6068](https://github.com/mruby/mruby/issues/6068) Stack overflow in mrb_vm_exec
+- [#6076](https://github.com/mruby/mruby/pull/6076) Fixed unwinding block that could point to invalid PC
+- [#6084](https://github.com/mruby/mruby/issues/6084) Incorrect symbolic sinks in binary built on Linux
 - [#6087](https://github.com/mruby/mruby/issues/6087) 'Remote branch HEAD not found in upstream origin' error on build
 - [#6089](https://github.com/mruby/mruby/issues/6089) binding.eval() handles def expressions differently from CRuby
 - [#6098](https://github.com/mruby/mruby/issues/6098) Fails to call superclass of wrapped method
+- [#6099](https://github.com/mruby/mruby/issues/6099) `ensure` section is not executed if the function exits via a return in a proc
 - [#6108](https://github.com/mruby/mruby/issues/6108) VM crashes with break
-- [#6134](https://github.com/mruby/mruby/issues/6134) String#unpack1 returns an array instead of a single string
-
-# Pull Requests (User Visible Ones)
-
-- [#5870](https://github.com/mruby/mruby/pull/5870) Use OP_LOADINEG instead of OP_LOADI
-- [#5966](https://github.com/mruby/mruby/pull/5966) Update default.gembox add mruby debugger mrdb
```

Comment:
Even though #5966 and other pull requests are mentioned above, we should list them here too, should we?